.github/workflows/versioning.yml

```
@@ -0,0 +1,42 @@
+name: Version Bump
+
+on:
+  push:
```

actionlint (1.7.12)

[error] "types" cannot be specified for "push" Webhook event

(events)